docs/reference/cluster-config.txt

```diff
@@ -451,6 +451,13 @@ The master supports the following configuration settings:
          instances. Defaults to the master instance ID if the master is
          on EC2, otherwise ``determined-ai-determined``.
 
+      -  ``custom_tags``: List of custom tags that are applied to the
```

Comment:
I think we should add one sentence clarifying what a custom tag is - i.e. it is an arbitrary user-defined tag that Determined will add to the agents, but it doesn't have any impact on how Determined works.